### PR TITLE
Add a role for bugs.cloud.gluster.org

### PR DESCRIPTION
**Pull request by mscherer**
So I am trying to see if we can move bugs.cloud.gluster to ansible. For now, that's a draft and would like people to review it, it is still missing a few stuff and currently run as root, which annoy me a bit. It also miss the cronjob in ndevos home that I need to port over.

So please do not merge for now :)
(also, i am out for 2 days, so there is no urgency, that's more a exploration on how we can handle PR and contribution, and so I tried to do that on github for that one )

Ping @nixpanic and @nigelbabu

### CODE REVIEW

File: roles/bugs/tasks/main.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+---
+- name: Install web configuration
+  template:
+    dest: /etc/nginx/conf.d/{{ server_name }}.conf
+    src: nginx.conf
+  notify: restart nginx
+
+- command: semanage fcontext -a -t httpd_user_rw_content_t /srv/{{ server_name }}/html/(/.*)?
```
Comment (nigelbabu) [inline roles/bugs/tasks/main.yml]: While this is obvious, it's useful to add `name:` here for better error output

File: roles/bugs/tasks/main.yml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+- name: Create the directory holding git checkout
+  file:
+    name: /srv/{{ server_name }}/html/
+    state: directory
+
+- name: Clone the repo
+  git:
+    repo: https://github.com/gluster/gluster-bugs-webui.git
+    dest: /srv/{{server_name }}/html/
+    version: HEAD
```
Comment (nigelbabu) [inline roles/bugs/tasks/main.yml]: `version: HEAD` is the default. It is reasonable to assume this is what we want without being explicit

**Comment by nigelbabu** [comment]
I think it's also useful to add README in every role folder explaining what the role is for. So far, it looks good to me.

**Comment by mscherer** [comment]
We miss the installation of various packages (like git, potentially python modules), for example.

File: roles/bugs/tasks/main.yml (hunk 2 above)
Comment (mscherer) [inline roles/bugs/tasks/main.yml]: Ok, removed it. I wonder if we want to do a automated cron upgrade, or if regular upgrade from running ansible every night will be sufficient.

**Comment by mscherer** [comment]
I am wondering if that's better to add the README in the role, or to just put that in the playbook. I also wonder if naming it "bugs" is a good idea, as I would call it nginx_bugs (since that's deploying nginx for bugs.gluster.org) to follow the naming convention I used for httpd_blog and others.

**Comment by mscherer** [comment]
Ok, so I will add some automated update. And I will also run that as non-root.

And I keep in mind that you do not like github for review (as said, that's just a test)

**Comment by mscherer** [comment]
So I pushed a few fix, if no one see anything wrong, i will merge after release.

**Comment by duck-rh** [comment]
I agree "bugs" may be a misleading name. "nginx_bugs" is too tied to a technology we could change anytime for technical reasons. So "website_bugs" or similar would be a better fit I think.

**Comment by duck-rh** [comment]
Why is bugs_username not in vars too?

Otherwise LGTM

**Comment by mscherer** [comment]
I used nginx_ because we also have others stuff using httpd. So I want later to be able to switch from one to the others, if we decide to only use httpd (or reinvent all the stuff done in nginx). But I forgot to merge, so i will do it now.

**Comment by duck-rh** [comment]
I though you would adapt the role instead of creating a new one/switching. Copying a configuration file is not really a big tie on a technology.

How about the variable?

**Comment by mscherer** [comment]
I am not even sure we need a variable. that's a rather crude role for a one off deployment.

**Comment by mscherer** [comment]
So, this got replaced by a jenkins jobs

**Comment by nixpanic** [comment]
On Thu, Jan 24, 2019 at 12:07:53AM -0800, Michael Scherer wrote:
> So, this got replaced by a jenkins jobs 

Thanks! I have disabled the cron-job that was running as my user.

It seems the last run of the Jenkins job failed:
  https://build.gluster.org/job/bugs-summary/62/console

I tried to have a look at the job and see how it works, but I can not
find it at https://github.com/gluster/centosci/tree/master/jobs. Where
is the job maintaineed?

**Comment by mscherer** [comment]
On https://review.gluster.org/#/admin/projects/build-jobs 

I guess the issue is some weird permission issue on the remote server side, I will try to fix it.

**Pull request closed without merging**